middlewares/profileValidation.js

```diff
@@ -0,0 +1,68 @@
+
+import validation from 'validator';
+
+const Categories = [
+  'Science',
+  'Technology',
+  'Engineering',
+  'Arts',
+  'Mathematic'
+];
+
+export default {
+
+  categoryValidator(req, res, next) {
+    if (Object.keys(req.body).includes('category')) {
+      if (!Categories.includes(req.body.category)) {
+        const error = new Error(
+          'Invalid category, Enter Science,Technology...');
```

Comment:
I think it's best to specify all the categories available here.

So your error message could be refined to something like:

`const allowedList = JSON.stringify(Categories);`
const errorMessage = `Invalid category. Allowed categories are ${allowedList}`
`const error = new Error(errorMessage)`

Reply:
Note that the error message should be wrapped with a template literal string.

Reply:
ok